### PR TITLE
Fix empty value pass to the client when complete() function not called

### PR DESCRIPTION
**Pull request by daneshk**
## Purpose
Fixes ballerina-platform/ballerina-standard-library#1152

## Checklist
- [x] Linked to an issue
- [x] Updated the changelog
- [x] Added tests

### CODE REVIEW

File: grpc-native/src/main/java/org/ballerinalang/net/grpc/callback/UnaryCallableUnitCallBack.java [hunk 1]
```diff
@@ -82,49 +82,51 @@ public void notifySuccess(Object response) {
             }
             return;
         }
+
+        Object content;
+        BMap headerValues = null;
+        if (isContextRecordByValue(response)) {
+            content = ((BMap) response).get(StringUtils.fromString("content"));
+            headerValues = ((BMap) response).getMapValue(StringUtils.fromString("headers"));
```
Comment (BuddhiWathsala) [inline grpc-native/src/main/java/org/ballerinalang/net/grpc/callback/UnaryCallableUnitCallBack.java:90]: Shall we make the "content" and "headers" constants?

**Comment by BuddhiWathsala** [comment]
Shall we add the respective proto file related to the 38th test?

File: grpc-native/src/main/java/org/ballerinalang/net/grpc/callback/UnaryCallableUnitCallBack.java (hunk 1 above)
Comment (daneshk) [reply grpc-native/src/main/java/org/ballerinalang/net/grpc/callback/UnaryCallableUnitCallBack.java]: Fixed in 23fbf72

**Comment by daneshk** [comment]
> Shall we add the respective proto file related to the 38th test?

Fixed in 23fbf72

**Review by BuddhiWathsala — APPROVED** [review]

**Pull request merged**